scripts/missions/cop/8_1_Garden_of_Antiquity.lua

```diff
@@ -0,0 +1,244 @@
+-----------------------------------
+-- Garden of Antiquity
+-- Promathia 8-1
+-----------------------------------
+-- !addmission 6 800
+-- Crystalline Field !pos .1 -10 -464 33
+-----------------------------------
+local altaieuID = zones[xi.zone.ALTAIEU]
+local palaceID  = zones[xi.zone.GRAND_PALACE_OF_HUXZOI]
+-----------------------------------
+
+local mission = Mission:new(xi.mission.log_id.COP, xi.mission.id.cop.GARDEN_OF_ANTIQUITY)
+
+local antiquityVars =
+{
+    -- [crystalOffset] = { thisFightDone, thisCsAcquired, otherCrystal, otherCrystal2, firstCsParam, secondCsParam1, secondCsParam2 }
```

Comment:
This should be replaced with missionStatus, as there are bits specific for this purpose:
>index 7 = 8-1 Rubious Crystals Bits
`player:setMissionStatus(COP, 6, x)`
```
0x2 - second rubious crystal
0x3 - third rubious crystal
```

Reply:
Just wanna note there's some minor issues with that documentation. There's a typo in that command (6 should be 7 since it's index 7) and the function signature has changed (7 and x swapped places). This mission does set index 6 to 0xF when you do the first crystal, then moves to index 7 for the last 2.